README.md

```diff
@@ -24,25 +24,38 @@ It contains a bare bones theme and must use plugin for you to base your developm
 		"mu-plugins/my-other-awesome-10up-plugin",
   ],
 ```
+
 6. To build plugins/themes simply run `npm install` at the root and `npm run [build|start|watch]` and npm will automatically build all themes and plugins. If a WordPress critical error is received run `composer install` in all locations that have an existing `composer.lock` file; example locations: `root`, `/mu-plugins/10up-plugin`, `/themes/10up-theme`. Upon build completion set the `10up-theme` as active within WordPress admin by running `10updocker wp theme activate 10up-theme`.
+
+ When running `composer install`, you may encounter the following warning:
+
+ ```bash
+ Warning: The lock file is not up to date with the latest changes in composer.json. You may be getting outdated dependencies. It is recommended  that you run `composer update` or `composer update <package name>`.
+ Your lock file does not contain a compatible set of packages. Please run composer update.
+ ```
+
+ To resolve this issue, simply execute the `composer update` command.
+
```

Comment:
**question:** Do we still need this addition now that you've updated the lock file?

Reply:
Absolutely! 😄 I totally considered that too! But I thought it'd be helpful to keep that addition for future reference, especially as new updates roll in. It can be a handy tip for fellow newbies like me. However, I do agree that the error itself is pretty self-explanatory, so it might be redundant to include it in the README. Let me know what you think.